### PR TITLE
fix: for helm issues

### PR DESCRIPTION
**Pull request by AlexsJones**
This correctly renders the chart from the release job and allows helm to be indexed and fetched from `helm repo add`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,6 +18,19 @@ The OpenFeature Operator is a Kubernetes native operator that allows you to expo
...
+## Helm
+
+```
+helm repo add openfeature https://open-feature.github.io/open-feature-operator/
+```
+
+```
+helm install ofo openfeature/ofo
+```
+
```
Comment (toddbaert) [inline README.md:30]: Should we perhaps also include:

`cd chart && helm install ofo .`?

**Review by toddbaert — APPROVED** [review]
This works beautifully to me and seems like a logical improvement over what we had before.

I tested this via a helm deploy on a clean cluster. I also tested the `make deploy-operator` installation and that worked as expected.

File: release-please-config.json [hunk 2]
```diff
@@ -9,6 +9,7 @@
       "extra-files": [
         "README.md",
         "chart/Chart.yaml",
+        "chart/values.yaml",
```
Comment (beeme1mr) [inline release-please-config.json:12]:
```suggestion
        "chart/ofo/values.yaml",
```

File: index.yaml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```
Comment (beeme1mr) [inline index.yaml:1]: Is this file automatically generated?

**Review by beeme1mr — COMMENTED** [review]
Can you git ignore the ofo tarballs?

**Comment by AlexsJones** [comment]
> Can you git ignore the ofo tarballs?

We need those so this repo can act as a helm server

File: index.yaml (hunk 3 above)
Comment (AlexsJones) [reply index.yaml:1]: Yes that's correct

File: release-please-config.json (hunk 2 above)
Comment (AlexsJones) [reply release-please-config.json:12]:
```
❯ ls chart
Chart.yaml	templates	values.yaml
```

This is the content of the chart, it doesn't have an `ofo` sub folder

Comment (toddbaert) [reply release-please-config.json:12]: I think the actual problem here is we might need `https://github.com/open-feature/open-feature-operator/pull/206/files#diff-cc5b1f6f50da569d760745a2cf39471cebd34940595c6c732e49cef8271a4030` (`charts/ofo/values.yaml`) added. It has a release-please annotation: https://github.com/open-feature/open-feature-operator/pull/206/files#diff-cc5b1f6f50da569d760745a2cf39471cebd34940595c6c732e49cef8271a4030R16

Comment (beeme1mr) [reply release-please-config.json:12]: Yeah, I was looking at this file.
https://github.com/open-feature/open-feature-operator/blob/bcc8c400ed8da97fa3aeb0a0ae8e8713581c7449/charts/ofo/values.yaml

**Review by james-milligan — APPROVED** [review]

File: release-please-config.json (hunk 2 above)
Comment (AlexsJones) [reply release-please-config.json:12]: Okay that's user error - that chart shouldnt be there, @james-milligan and I were debugging. MY BAD!

**Comment by beeme1mr** [comment]
> We need those so this repo can act as a helm server

@AlexsJones are those tar files committed to the repo as part of the release process?

**Comment by AlexsJones** [comment]
> > We need those so this repo can act as a helm server
> 
> @AlexsJones are those tar files committed to the repo as part of the release process?

They are also for posterity so users can find old versions.

Here is an example...
```
❯ helm search repo ofo
WARNING: Repo "kube-vip" is corrupt or missing. Try 'helm repo update'.
WARNING: open /Users/alexjones/Library/Caches/helm/repository/kube-vip-index.yaml: no such file or directory
NAME   	CHART VERSION	APP VERSION	DESCRIPTION
ofo/ofo	v0.2.6       	v0.2.6     	A feature flag operator for Kubernetes
ofo/ofo	v0.2.7       	v0.2.7     	A feature flag operator for Kubernetes
ofo/ofo	v0.2.7       	v0.2.8     	A feature flag operator for Kubernetes

```

_These are an example_

**Pull request merged**